src/osd/SnapMapper.cc

```diff
@@ -972,4 +992,65 @@ int SnapMapper::convert_legacy(
   }
   return 0;
 }
+
+
+int SnapMapper::convert_malformed(
+  CephContext *cct,
+  ObjectStore *store,
+  ObjectStore::CollectionHandle& ch,
+  ghobject_t hoid,
+  unsigned max)
```

Comment:
please don't use 'max':
a) it's not descriptive enough, and
b) greps with the max() function/macro